packages/strapi-plugin-upload/services/Upload.js

```diff
@@ -77,7 +77,11 @@ module.exports = {
     // Execute upload function of the provider for all files.
     return Promise.all(
       files.map(async file => {
-        await actions.upload(file);
+        
+        // This should host the provider configuration
+        const options = {};
```

Comment:
The options are always an empty object. how should that work ?

Reply:
By filling it as explained in the PR description.

Reply:
This should come from the configuration not a codebase modification

Reply:
Can you please point at this configuration so maybe I can add it in there?